config/locales/vi.yml

```diff
@@ -0,0 +1,182 @@
+---
+vi:
+  activerecord:
+    errors:
+      messages:
+        record_invalid: "Có các lỗi sau: %{errors}"
+        restrict_dependent_destroy:
+          has_one: Không thể xóa do tồn tại đối tượng phụ thuộc %{record}
+          has_many: Không thể xóa do tồn tại một số đối tượng phụ thuộc %{record}
+  date:
+    abbr_day_names:
+      - CN
+      - Thứ 2
+      - Thứ 3
+      - Thứ 4
+      - Thứ 5
+      - Thứ 6
+      - Thứ 7
+    abbr_month_names:
+      -
+      - Thg 1
+      - Thg 2
+      - Thg 3
+      - Thg 4
+      - Thg 5
+      - Thg 6
+      - Thg 7
+      - Thg 8
+      - Thg 9
+      - Thg 10
+      - Thg 11
+      - Thg 12
+    day_names:
+      - Chủ nhật
+      - Thứ hai
+      - Thứ ba
+      - Thứ tư
+      - Thứ năm
+      - Thứ sáu
+      - Thứ bảy
+    formats:
+      default: "%d-%m-%Y"
+      long: "%d %B, %Y"
+      short: "%d %b"
+    month_names:
+      -
```

Comment:
remove this line

Reply:
This is for month zero, which does not exist. ✌🏽

Reply:
The same goes for the lines above.

Reply:
what is month zero?

Reply:
@nguyenphuhongloan 
It does not exist. This line is for better mapping between a month and its locale, e.g., month 1 for Jan. So we do not have to subtract 1 from the month.